WooCommerce/Classes/ViewRelated/Coupons/Add and Edit Coupons/AddEditCoupon.swift

```diff
@@ -6,7 +6,7 @@ final class AddEditCouponHostingController: UIHostingController<AddEditCoupon> {
 
     private let viewModel: AddEditCouponViewModel
 
-    init(viewModel: AddEditCouponViewModel, onDisappear: @escaping () -> Void) {
+    init(viewModel: AddEditCouponViewModel, onDisappear: @escaping (_ success: Bool) -> Void) {
```

Comment:
The `success` parameter makes sense in the context of this PR, but if I saw it out of the blue it might be tricky to understand what's the success referring to, like "success of disappear"? Would it make sense to rename the param to something like `isCouponCreationSuccess`, `couponCreateResultCallback`, or similar?

Reply:
Thanks. If question like this comes up, then it definetely makes sense to clarify it 👍